### PR TITLE
[libc++] Test GitHub PRs for libc++ patches

### PR DESCRIPTION
**Pull request by ldionne**

### CODE REVIEW

**Review by SatinWukerORIG — APPROVED** [review]

**Review by StepfenShawn — APPROVED** [review]

**Review by xgupta — APPROVED** [review]

**Review by bitcoinfinancier — APPROVED** [review]

**Comment by greg7mdp** [comment]
Can I submit a libc++ PR via github?

**Review by VoltrexKeyva — APPROVED** [review]

**Review by poyaoc97 — APPROVED** [review]
LGTM

**Comment by ldionne** [comment]
> Can I submit a libc++ PR via github?

Not yet, we're working on that!

I'm going to lock this issue as I pick this up again and start experimenting with the GH action configuration necessary to make this work. I suggest folks unsubscribe from this PR because this will get quite spammy :)

**Comment by ldionne** [comment]
I am waiting for an admin to add the BuildKite API token I provided to the repo :-)

File: .ci/monolithic-linux.sh [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+MONOREPO_ROOT="${MONOREPO_ROOT:="$(git rev-parse --show-toplevel)"}"
+BUILD_DIR="${BUILD_DIR:=${MONOREPO_ROOT}/build/monolithic-linux}"
+
+rm -rf ${BUILD_DIR}
+
+ccache --zero-stats
+ccache --show-config
+
+pip install -q -r ${MONOREPO_ROOT}/mlir/python/requirements.txt
```
Comment (tstellar) [inline .ci/monolithic-linux.sh:27]: Test Comment.

**Comment by ldionne** [comment]
Closing this since cf1a3d93581f4c4698accd4418a15399d8814bb8 was merged via Phabricator.

CI should work now for PRs but we'll probably need to make some tweaks.

**Pull request closed without merging**